scripts/kind-e2e/e2e.sh

```diff
@@ -363,84 +228,64 @@ logging=$3
 kubefed=$4
 deploy=$5
 
+echo Starting with status: $status, k8s_version: $version, logging: $logging, kubefed: $kubefed, deploy: $deploy
+
 if [[ $status = clean ]]; then
     cleanup
     exit 0
 fi
 if [[ $status != keep && $status != create ]]; then
+    # FIXME: Does not cover status=onetime, which is used below
```

Comment:
I suspect for onetime we didn't want a cleanup (I don't remember the purpose of onetime though)

Reply:
onetime is the default, ie run the tests and clean up.

Reply:
Ah, I see what it was meant to to do now, thanks. Refactored to make more clear in the latest version.